connections/etuovi/services.py

```diff
@@ -38,22 +37,22 @@ def fetch_apartments_for_sale() -> list:
     return items
 
 
-def create_xml(items: list) -> Tuple[str, str]:
+def create_xml(items: list) -> str:
```

Comment:
Since this function can also return `None` I think the type should be `Optional[str]`.

Reply:
Yes, again same issue :) Updating...